Storage/test.cc

```diff
@@ -184,6 +186,40 @@ TEST(TransactionalStorage, SmokeTest) {
       }).Go();
       EXPECT_TRUE(WasCommitted(result));
     }
+    {
+      const auto result = storage.Transaction([](MutableFields<Storage> fields) {
+        EXPECT_TRUE(fields.o.Empty());
+        EXPECT_EQ(0u, fields.o.Size());
+        EXPECT_TRUE(fields.o.Rows().Empty());
+        EXPECT_TRUE(fields.o.Cols().Empty());
+        fields.o.Add(Cell{1, "one", 1});
+        fields.o.Add(Cell{2, "two", 2});
+        fields.o.Add(Cell{2, "too", 3});
+        fields.o.Add(Cell{3, "too", 4});
+        EXPECT_FALSE(fields.o.Empty());
+        EXPECT_EQ(2u, fields.o.Size());
```

Comment:
And as `OneToOne` overwrites are a bit less obvious, some comments explaining what's going on here could increase readability :)